Orange/widgets/evaluate/owparameterfitter.py

```diff
@@ -54,8 +55,8 @@ def _search(
         data: Table,
         learner: Learner,
         fitted_parameter_props: Learner.FittedParameter,
-        initial_parameters: dict,
-        steps: Sized,
+        initial_parameters: dict[str, Any],
```

Comment:
Currently, we'd have `dict[str, int]`, but eventually we'd have `dict[str, Union[int, float]]`. On the other hand, what if the learner provides a list of string arguments? So I put `dict[str, Any]`. We can put a humble `int` there, though, for the time being.